code/datums/traits/neutral.dm

```diff
@@ -168,3 +168,16 @@
 		SEND_SIGNAL(H.back, COMSIG_TRY_STORAGE_SHOW, H)
 
 	to_chat(quirk_holder, "<span class='notice'>There is a pinpointer [where], which can help you find your way around. Click in-hand to activate.</span>")
+
+// fluff/desc/implementation/everything is WIP
+/datum/quirk/assistant_hater
+	name = "Aporophobe"
+	desc = "You have such a deep seated aversion to assistants that you had an illegal neural blocker installed so that you couldn't see them anymore."
```

Comment:
*deep-seated

also, knowing Nanotrasen, these implants would totally be legal for use in NT facilities. hell, I bet half of the cod- er, upper brass of Nanotrasen have assistant-blocking implants installed in them.